docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -1076,3 +1076,7 @@ keep a backup of the original values if you change them.
     catalog specific use.
   - `false`
 :::
+
+### Object storage filesystem caching
```

Comment:
nit: I would just call it "File system cache" or "Data files cache". It's not necessary for it to be an object store, it could work with a hierarchical file system like HDFS too.

Reply:
ok ... will use Filesystem cache